src/reducers/index.js

```diff
@@ -0,0 +1,4 @@
+import { combineReducers } from 'redux';
+export default combineReducers({
+
+});
```

Comment:
Nhớ luôn bỏ một xuống hàng ở mỗi file
Khi thêm nội dung mới -> git nó hiểu là thêm nội dung mới

Chứ không lần sau xuống hàng, thì git nó hiểu là mình có chỉnh sửa ở dòng `}); ` -> git diff không được đẹp

```suggestion
});

```